.github/workflows/doxygen.yml

```diff
@@ -15,16 +15,33 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
 
+      - name: Create documentation branch
+        run: git checkout -b documentation/$GITHUB_SHA
+
       - name: Generate Documentation
         run: scripts/cibuild.sh --documentation -DDOXYGEN_OUTPUT_DIR=$GITHUB_WORKSPACE/novelrt/docs
 
+      #Mandatory step it appears
+      #Username and Email found from https://github.community/t/github-actions-bot-email-address/17204/3
+      - name: Configure Git
+      - run: |
+          git config user.name "github-actions"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
In reference to this convo: https://github.com/novelrt/NovelRT/pull/257/files/046abc10878bc4185964794d304d061ccd4bda8a#r610989328